header.html

```diff
@@ -41,96 +41,99 @@ <h1>I’m Mariane
-                        <li class="no-count">CANOPY</li>
-                        <li>Back End Dev</li>
-                        <li>2015</li>
-                    </ul>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                    <ul class="card-tags">
-                        <li><a href="">html</a></li>
-                        <li><a href="">css</a></li>
-                        <li><a href="">javascript</a></li>
-                    </ul>
-                    <button class="card-button" type="submit">See Project</button>
-                </div>
-            </section>
-            <!--Work Cards 4-->
-            <section class="card">
-                <img src="img/Snapshoot-Portfolio3.svg" alt="Snapshoot-Portfolio">
-                <div class="card-block">
-                    <h2>Multi-Post Stories</h2>
-                    <ul class="frame">
-                        <li class="no-count">CANOPY</li>
-                        <li>Back End Dev</li>
-                        <li>2015</li>
-                    </ul>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                    <ul class="card-tags">
-                        <li><a href="">html</a></li>
-                        <li><a href="">css</a></li>
-                        <li><a href="">javascript</a></li>
-                    </ul>
-                    <button class="card-button" type="submit">See Project</button>
-                </div>
-            </section>
-            <section class="about">
+        <section class="card">
+            <img src="img/Snapshoot-Portfolio.svg" alt="Snapshoot-Portfolio">
+            <div class="card-block">
+                <h2>Tonic</h2>
+                <ul class="card-frame">
+                    <li class="no-count">CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="card-tags">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <button class="card-button" type="submit">See Project</button>
+            </div>
+        </section>
+        <!--Work Cards 2-->
+        <section class="card card-reverse">
+            <img src="img/Snapshoot-Portfolio1.svg" alt="Snapshoot-Portfolio" class="float-img">
+            <div class="card-block">
+                <h2>Multi-Post Stories</h2>
+                <ul class="card-frame">
+                    <li class="no-count">CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```

Comment:
- [ ] Kindly note that you are required to stick as much as possible to the provided Figma design in fonts, colors, etc. The font size for this p should be 16px as specified in the Figma design.

**Kindly implement this on all the ```<p>``` for the works section**